packages/table/src/Table.tsx

```diff
@@ -21,6 +21,7 @@ export const Table = forwardRef<HTMLTableElement, TableProps>(function Table(
 			fontSize="sm"
 			focus
 			width="100%"
+			display="table"
```

Comment:
display: table is implied by using a Table HTML element, but I figured it would be good for consistency to add